src/neuroconv/tools/spikeinterface/spikeinterface.py

```diff
@@ -664,6 +661,11 @@ def add_electrical_series(
         else:
             eseries_kwargs.update(conversion=1e-6)
             eseries_kwargs.update(channel_conversion=channel_conversion)
+
+    # Iterator
+    if iterator_opts is None:
+        iterator_opts = dict()
```

Comment:
You can also do this kind of thing as `iterator_opts = iterator_opts or dict()` to save a line. We also usually do that kind of argument parsing as early in the function as possible (right after header).